api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1875,7 +1874,7 @@ public void makeBucket(String bucketName, String region)
       configString = config.toString();
     }
 
-    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, 0);
+    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, configString.length());
```

Comment:
`String.length()` != `[].byte.length`.

We would need to convert string into UTF8 encoded byte array.